README.md

```diff
@@ -1,11 +1,11 @@
-![360Giving Logo](http://openphilanthropy.files.wordpress.com/2014/02/cropped-360givinglogo-2010-size.jpg)
+![360Giving Logo](./360standard-color.png)
 
 360Giving Standard
 ========
 
 360Giving aims to help UK grant makers and philanthropists to publish their grant information online in an easy to use way.  We are motivated by a commitment to the value of open data and strategic funding.
 
-The **360Giving Data Standard** provides a common approach for publishing grant data. 
+The **360Giving Data Standard** provides a common approach for publishing grant data.
 
 For more about the overall project, visit the main [360Giving website](http://threesixtygiving.com/about/).
```

Comment:
update to current website url: https://www.threesixtygiving.org/

Reply:
also be good to update url on row 12 from http://www.threesixtygiving.org/standard to https://standard.threesixtygiving.org/en/latest/